lib/experimental/fonts/font-library/font-library.php

```diff
@@ -132,3 +132,22 @@ function wp_get_font_dir( $defaults = array() ) {
 		return apply_filters( 'font_dir', $defaults );
 	}
 }
+
+// @core-merge: This code should probably go into ...?
```

Comment:
I'm not sure, but I wanted to try to help! 😅 Maybe in [wp-includes/block-editor.php](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/block-editor.php)? As `use_block_editor_for_post filter` is defined in [wp-includes/post.php](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/post.php#L8066) which is nearby.